hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -494,6 +494,11 @@ private OMConfigKeys() {
 
   public static final String OZONE_OM_SNAPSHOT_DIFF_REPORT_MAX_PAGE_SIZE
       = "ozone.om.snapshot.diff.max.page.size";
+
+  public static final String OZONE_OM_SNAPSHOT_DB_MAX_OPEN_FILES
+      = "ozone.om.snapshot.db.max.open.files";
+  public static final int OZONE_OM_SNAPSHOT_DB_MAX_OPEN_FILES_DEFAULT
```

Comment:
QQ: Should we use -1 here just to make sure if it is not set -1 is used?

Reply:
It is better to have a proper limit? Instead of having no limit, otherwise there would be more failures & more tickets going forward for this.